packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFormat.ts

```diff
@@ -66,7 +66,8 @@ type EncodedNodeExistsConstraint = Static<typeof EncodedNodeExistsConstraint>;
 export const EncodedNodeChangeset = Type.Object(
 	{
 		fieldChanges: Type.Optional(EncodedFieldChangeMap),
-		nodeExistsConstraint: Type.Optional(EncodedNodeExistsConstraint),
+		inputNodeExistsConstraint: Type.Optional(EncodedNodeExistsConstraint),
+		outputNodeExistsConstraint: Type.Optional(EncodedNodeExistsConstraint),
```

Comment:
Our changeset parsers are configured to break when they encounter data that they don't know of.
The introduction of `inputNodeExistsConstraint` would make newer (i.e., after this change) versions of the code break when trying to open old documents or when collaborating with peers that run older (i.e., before this change) versions of the code.
The introduction of `outputNodeExistsConstraint` would make older (i.e., before this change) versions of the code break when trying to open new documents or when collaborating with peers that run newer (i.e., after this change) versions of the code.

In order to make this change is a safe way, we would need to create a new version of the ModularChangeset schema and it can be used safely.
I don't think we want to do that just yet: the support for undo constraints is somewhat urgent, and we won't get any benefit from having this data being sent over the wire since we don't allow peers to undo one-another's changes.

Reply:
Reverted the renaming of the existing constraint and removed the new one from the codecs so that they are not sent over the wire.